### DISCUSSION TITLE
`BeatSyncedContainer` randomly stops animating itself

### DISCUSSION THREAD

**Discussion by Feodor0090** [ppy/osu#21062] [Q&A] [answered]
It happens with all beatmaps randomly. Just switch them and at some point of time it will stop animating. Notice not blinking search textbox cursor on 0:03 and that beat samples are not played when i hover osu! logo.

Running 2022.1031.0, but it can be reproduces in older versions. Debian 11 with PulseAudio, if it matters.

https://user-images.githubusercontent.com/53872073/199286065-cefa96e6-3f03-4139-a218-f6d58052f27a.mp4

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/21069
